sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -521,50 +646,82 @@ def _to_internal_entity(self):
         return self._internal_sd
 
 
-class SubscriptionRuntimeInfo(object):
-    """Description of a Service Bus queue resource.
+class SubscriptionRuntimeProperties(object):
+    """Runtime properties of a Service Bus topic subscription resource.
 
     :ivar str name:
-    :ivar created_at: The exact time the queue was created.
+    :ivar created_at: The exact time the subscription was created.
     :type created_at: ~datetime.datetime
     :ivar updated_at: The exact time a message was updated in the queue.
     :type updated_at: ~datetime.datetime
     :ivar accessed_at: Last time a message was sent, or the last time there was a receive request
```

Comment:
have we ever discussed about whether we should "_utc" suffix on the mgmt time related properties?

Reply:
Created issue #12900 to follow up. This needs discussion with other languages and other Python SDKs